openmetadata-ui/src/main/resources/ui/src/components/Alerts/AlertDetails/AlertDiagnosticInfo/AlertDiagnosticInfoTab.interface.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ *  Copyright 2024 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+export interface AlertDiagnosticData {
+  latestOffset: number;
+  currentOffset: number;
+  startingOffset: number;
+  hasProcessedAllEvents: boolean;
+  successfulEventsCount: number;
+  failedEventsCount: number;
+  relevantUnprocessedEventsCount: number;
+  totalUnprocessedEventsCount: number;
+}
+
+export interface AlertDiagnosticInfoTabProps {
+  diagnosticData: AlertDiagnosticData;
+}
```

Comment:
Let's use existing types from `generated` folder. We don't need to create new types for this.